source/includes/windows/common-minio-kes.rst

```diff
@@ -141,6 +142,14 @@ MinIO uses the :envvar:`MINIO_KMS_KES_KEY_NAME` key for the following cryptograp
   include a specific |EK|.
 - Encrypting objects using :ref:`SSE-S3 <minio-encryption-sse-s3>`.
 
+MinIO uses the :envvar:`MINIO_KMS_KES_ENCLAVE` key to define the name of the KES enclave to use.
+
+- Replace ``<name>`` with the name of the :term:`enclave` to use.
+- If not defined, MinIO does not send any enclave information.
+  This may result in using the default enclave for stateless KES servers.
```

Comment:
```suggestion
  This may result in using the default enclave for stateful KES servers.
```

Reply:
I'd rather have something in the docs, since people will see it in the Github change notes as a new envvar.